src/comparator/wrappers.py

```diff
@@ -35,8 +35,8 @@
 
 
 def _get_source_code_line(source_code_locations, path):
-    if path not in source_code_locations:
-        return f"No source code line can be identified by path {path}."
+    if not source_code_locations or path not in source_code_locations:
+        return -1
```

Comment:
Return message is changed from a "str" to "-1" because when outputing the findings, we will sort the line number based on "_get_source_code_line" method. "str" will mess up the sorting and break the tool.